cli/test-resources/projects/tasks-plugins-sdk/portlets/tasks-portlet/docroot/WEB-INF/service/com/liferay/tasks/exception/NoSuchTasksEntryException.java

```diff
@@ -22,7 +22,7 @@
 /**
  * @author Brian Wing Shun Chan
  */
-public class NoSuchTasksEntryException extends NoSuchModelException {
+public class NoSuchTasksEntryException exceptionxtends NoSuchModelException {
```

Comment:
We need to add everything in test-resources to "dont format"

Reply:
haha okay. i'm not done with the SF. my eyes are O_o

Reply:
just wanted to push it up in case comp crashes or something crazy